.gitignore

```diff
@@ -14,3 +14,11 @@ cmake-build-*/
 .idea/
 .idea
 
+#Conan created files
+CMakeUserPresets.json
+
+#Compilation database used by lsp
+compile_commands.json
```

Comment:
what's lsp?

Reply:
https://github.com/emacs-lsp/lsp-mode/discussions/3478 maybe?

Reply:
Yep, Language Server Protocol - https://en.wikipedia.org/wiki/Language_Server_Protocol & https://neovim.io/doc/user/lsp.html

In this case the file is used by [clangd](https://clangd.llvm.org/) so that may be a better name to use in the comment.